tests/Integration/Frameworks/Laravel/Version_4_2/composer.json

```diff
@@ -25,6 +25,7 @@
 			"DDTrace\\": "../../../../../src/DDTrace/"
 		},
 		"files": [
+			"../../../../../src/DDTrace/Formats.php",
```

Comment:
😢 would be great to find some approach - not requiring this.
But this is out of scope of this PR.

Reply:
I know :(